docs/provisioner/01-01-provisioner-overview.md

```diff
@@ -3,19 +3,15 @@ title: Overview
 type: Overview
 ---
 
-The Runtime Provisioner is a Compass component responsible for provisioning, installing, and deprovisioning clusters with Kyma (Kyma Runtimes). The relationship between clusters and Runtimes is 1:1.
+The Runtime Provisioner is a Compass component responsible for provisioning, installing, and deprovisioning clusters with Kyma (Kyma Runtimes). The relationship between clusters and Runtimes is 1:1. The Runtime Provisioner is registered in the Director as an integration system. 
 
 It is powered by [Hydroform](https://github.com/kyma-incubator/hydroform) and it allows you to provision the clusters in the following ways:
 - [through Gardener](08-02-provisioning-gardener.md) on:
     * GCP
     * Microsoft Azure
     * Amazon Web Services (AWS).
     
-Note that the operations of provisioning and deprovisioning are asynchronous. They return the operation ID, which you can use to [check the Runtime Operation Status](08-03-runtime-operation-status.md).
-
-The Runtime Provisioner also allows you to [clean up Runtime data](08-06-clean-up-runtime-data.md). This operation removes the data for a given Runtime from the database and frees up the Runtime ID for reuse. It is useful when your cluster has died or when the operation of deprovisioning has failed.
-
-> **CAUTION:** The operation of cleaning up Runtime data does not trigger Runtime deprovisioning and the cluster might still exist after running it.
+Note that the operations of provisioning and deprovisioning are asynchronous. The operation of provisioning returns the Runtime Operation Status containing the runtime ID and the operation ID. The operation of deprovisioning returns the operation ID. You can use the operation ID to [check the Runtime Operation Status](08-03-runtime-operation-status.md).
```

Comment:
```suggestion
Note that the operations of provisioning and deprovisioning are asynchronous. Provisioning returns the Runtime Operation Status that contains the Runtime ID and the operation ID. Deprovisioning returns the operation ID. You can use the operation ID to [check the Runtime Operation Status](08-03-runtime-operation-status.md).
```

Reply:
Consider putting that info in a **NOTE**.

Reply:
I didn't cos there are already other notes and on top of looking ugly that would probably disappear amongst them.